aiohttp/multipart.py

```diff
@@ -195,21 +194,26 @@ def content_disposition_filename(params: Mapping[str, str],
 
 
 class MultipartResponseWrapper:
-    """Wrapper around the MultipartBodyReader.
+    """Wrapper around the MultipartReader.
```

Comment:
There is no such of thing as `MultipartBodyReader` so decided to go with `MultipartReader` as the only thing that matched the interface. I will give it another look, but for now that was the thing that matched.